scripts/build_scripts/build_zips.py

```diff
@@ -167,14 +186,19 @@ def get_targets_args(targets):
       camke args included targets.
   """
   result_args = []
+  
+  support_targets = SUPPORT_TARGETS
+  if is_tvos_build() and not targets:
+    targets = TVOS_SUPPORT_TARGETS
```

Comment:
For tvOS do you also want to set support_targets to TVOS_SUPPORT_TARGETS, to catch if someone tries to run: --platform=tvos --apis=messaging
?